### ISSUE TITLE
Is this deprecated?

### ISSUE THREAD

**Issue by KyleAMathews** [rvagg/node-slow-stream#1]
Just tried using it and nothing seemed to happen...

**Comment by rvagg** [comment]
yeah, 3 years old, I used it for some tests once but don't think I use it anywhere any more, probably look at one of the alternatives, this one doesn't even do streams2

**Comment by saper** [comment]
any particular alternatives to try?
